src/api/app/components/add_review_collapsible_component.html.haml

```diff
@@ -0,0 +1,17 @@
+.card.collapse.position-absolute.w-100.mt-2.shadow-lg{ id: 'review_form_collapse' }
+  .card-body
+    %h5
+      Add a review for
+      %i
+    = form_tag(request_modify_review_path) do
+      = hidden_field_tag('review_id')
+      .form-group
+        = text_area_tag('comment', '', rows: 4, class: 'w-100 form-control', placeholder: 'Please comment on your decision')
+      .form-group
+        .custom-control.custom-radio
+          = radio_button_tag('new_state', 'Decline', true, class: 'custom-control-input')
+          = label_tag('Decline', 'Decline', for: 'new_state_Decline', class: 'custom-control-label')
+        .custom-control.custom-radio
+          = radio_button_tag('new_state', 'Approve', false, class: 'custom-control-input')
+          = label_tag('Approve', 'Approve', for: 'new_state_Approve', class: 'custom-control-label')
```

Comment:
I'm missing some form of information what exactly those two actions do.

Reply:
I added some description